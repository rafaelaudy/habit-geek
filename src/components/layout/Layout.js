import React, { Fragment } from "react";
import Header from "./Header";

function Layout({ children }) {
  return (
    <Fragment>
      <Header />
      {children}
    </Fragment>
  );
}

export default Layout;
