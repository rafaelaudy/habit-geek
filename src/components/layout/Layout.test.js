import React from "react";
import { shallow } from "enzyme";
import Layout from "./Layout";

describe("Layout component", () => {
  it("Renders static elements", () => {
    const layout = shallow(<Layout />);
    expect(layout).toMatchSnapshot();
  });
});
