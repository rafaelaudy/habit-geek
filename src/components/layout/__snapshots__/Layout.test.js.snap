// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component Renders static elements 1`] = `
<Fragment>
  <Header />
</Fragment>
`;
